omnibus/config/software/datadog-dogstatsd.rb

```diff
@@ -18,7 +18,7 @@
   gopath = Pathname.new(project_dir) + '../../../..'
   env = {
```

Comment:
I'm surprised we don't use `with_embedded_path` here, but that was pre-existent (and if I'm being honest I'm not quite convinced `with_embedded_path` serves much of a purpose now that we don't build `pkg-config`

Reply:
where's your adventurous spirit :D